ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/useClusterExtendedDetails.ts

```diff
@@ -22,17 +32,23 @@ const clusterExtendedDetailsQuery = gql`
     }
 `;
 
+export type ProviderMetadata = {
+    aws: Record<string, unknown> | null;
+    azure: Record<string, unknown> | null;
+    google: Record<string, unknown> | null;
+    region: string;
+};
+
```

Comment:
Using the restrictive type of `Record<string, unknown>` here since we are not querying any fields for these objects, we only need to know if the object itself is non-null to determine the cloud provider.